datus-redshift/datus_redshift/connector.py

```
@@ -0,0 +1,1345 @@
+            params: Optional query parameters
+            
+        Returns:
+            Pandas DataFrame with query results
+        """
+        try:
+            # Get Arrow table first
+            arrow_table, _ = self._do_execute_arrow(sql, params)
+            
+            # Convert Arrow to pandas DataFrame
+            return arrow_table.to_pandas()
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)